### ISSUE TITLE
Gnonsis Safe Support

### ISSUE THREAD

**Issue by ryley-o** [manifoldxyz/royalty-registry-client#12]
Supporting Gnosis Safe wallets on the frontend would be very helpful.

Generally, it is best-practice for the owner/admin of a given contract to be using a multi-sig wallet, such as Gnosis Safe. Thus, a common use-case when assigning overrides for a contract would be to connect to the Royalty Registry frontend with a Gnosis Safe.

An additional driver: The fact that the registry uses a proxy pattern makes it quite difficult for anyone non-technical to properly configure their override via contract calls directly. Knowledge of which contract to integrate with, and which ABI to use is critical when not using the frontend.

**Comment by jakerockland** [comment]
Yes please 🙏 😄 This would be very appreciated for us at Art Blocks but I think is also aligned with general opsec best practices in terms of protocol-managed/team-managed contracts that are owned by a multi-sig rather than a single EOA.

**Comment by wwhchung** [comment]
I support this.  The code is open source, so... I'm happy to merge in whatever code is added to make this possible.

In the meantime, I guess, maybe some instructions on how to do it manually would help?  I would think ppl using Gnosis for contract interactions in general would have some technical competance?

**Comment by wwhchung** [comment]
The code is around here, where we currently just use window.ethereum:
https://github.com/manifoldxyz/royalty-registry-client/blob/aaf22c42671512a61e8d4e659c00ef06ab2e8c5d/src/App.vue#L54
